setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='hapipy',
-    version='2.10.5.4',
+    version='2.10.5.5',
```

Comment:
I would suggest making the new version `2.10.5.4-bs1` instead of utilizing a version number that could exist outside our walls.

Reply:
After all, this PR is a big change. How about making it as `2.10.5.5-bs1`?

Reply:
Or actually, 2.10.6-bs1 should be good for this Python2/3 compatibility change?

Reply:
If you were looking at the requirements file and saw 2.10.5.5-bs1, what version of the library would you think our changes were based on?  I'm guessing you'd be really confused when you looked at the upstream project and saw that 2.10.5.4 was the newest version.

When you do an internal version number like this, you should retain the upstream's version number that you branched from and then extend it, like we do by adding `-bs1` to the version.